src/Template/Reviews/of.ctp

```diff
@@ -133,7 +146,9 @@ $this->set('title_for_layout', $this->Pages->formatTitle($title));
             $withAudio = false;
             foreach ($corpus as $item) {
                 $sentence = $item->sentence;
-                echo '<div>';
+                $sentenceData = $this->Sentences->sentenceForAngular($sentence);
+                $sentenceData = str_replace('{{', '\{\{', $sentenceData);
```

Comment:
Same here: I don’t think we need these two lines any more.

Reply:
Again, good catch.